### PR TITLE
rwd: added chapter 6

### PR DESCRIPTION
**Pull request by LordRafa**

### CODE REVIEW

File: chapter06/rwd/ex3-6.pl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my $max = 0;
+foreach my $key (keys %ENV) {
+  ($max > length($key)) || ($max = length($key));
```
Comment (codavex) [inline chapter06/rwd/ex3-6.pl:8]: Nice!

File: chapter06/rwd/ex1-6.pl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my %surnames = ( fred => "flintstone", barney => "rubble", wilma => "flintstone" );
+
+print "Type a name to find the surname: ";
+chomp(my $name= <STDIN>);
+print( "Surname: " . ($surnames{$name} // "WOPS!!! No surname found.") . "\n" );
```
Comment (codavex) [inline chapter06/rwd/ex1-6.pl:10]: might be cool to mention // in the meeting

Comment (qianliok) [inline chapter06/rwd/ex1-6.pl:10]: Yes, it looks neat. I assume `//` will handle the non exists hash key/value.

File: chapter06/rwd/ex2-6.pl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+my %words;
+
+print "Type words, one per line (CTRL-D to stop):\n";
+while (<STDIN>) {
+	chomp;
+	$words{$_}++;
+}
+
+print Data::Dumper->Dump([\%words], [qw(Sumary)]);
```
Comment (qianliok) [inline chapter06/rwd/ex2-6.pl:15]: will it dump in order?

Comment (LordRafa) [inline chapter06/rwd/ex2-6.pl:15]: wopsss I suppose that If I would sort it yes, but I didn't knew that it had to be sorted '^^

**Pull request merged**